bdio2/src/main/java/com/blackducksoftware/bdio2/LegacyScanContainerEmitter.java

```diff
@@ -404,7 +404,10 @@ private Product scanClient() {
                     .addCommentText("signature %s", signatureVersion);
             if (ownerEntityKeyToken != null && ownerEntityKeyToken.startsWith("SP#")) {
                 scanClient.addCommentText("snippets");
+            } else if (ownerEntityKeyToken != null && ownerEntityKeyToken.startsWith("SS#")) {
```

Comment:
I wouldn't use "SS" as this is already reserved for "EVENT_SUBCRIPTION_TYPE" in our entity types.

Reply:
Changed to 'SG'. It is not used. The logic behind the name is that S and G are first and last letters of the 'string', something that associates with string ))